File: backend/app/Models/Task.php
<?php

namespace App\Models;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\BelongsToMany;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Task extends Model
{
    public $timestamps = false;

    protected $fillable = [
        'quiz_id',
        'order',
        'header',
        'text',
    ];

    public function quiz() : BelongsTo
    {
        return $this->belongsTo(Quiz::class);
    }

    public function subtasks() : HasMany
    {
        return $this->hasMany(Subtask::class);
    }
}

File: tests/BolcsessegBarkaAPITests/ApiLibrary/Task.cs
using Newtonsoft.Json;

namespace ApiLibrary;

public class Task
{
    [JsonProperty("id")]
    public int Id { get; set; }
    
    [JsonProperty("order")]
    public int Order { get; set; }
    
    [JsonProperty("header")]
    public string Header { get; set; }
    
    [JsonProperty("text")]
    public string Text { get; set; }
    
    [JsonProperty("subtasks")]
    public List<Subtask> Subtasks { get; set; }
}